source/tools/entry.js

```diff
@@ -6,6 +6,16 @@ const EntryProperty = {
     Username:                     "username"
 };
 
+const EntryDefaultAttribute = {
+    PassModifiedTime:              "passModifiedTime",
+    PassExpiryTime:                "passExpiryTime",
+    PassExpiry:                    "passExpiry"
```

Comment:
Could you perhaps explain why we need 3 parameters? For instance, don't we just care about the following?

 * Last time the password was changed
 * How long until the password expires

Come to think of it.. wouldn't the expiry be best suited as a number representing seconds (perhaps?) until the password expires? That way we'd only store 2 values. Juggling 3 will make error correction a pain in the a 😅